src/com/lftechnology/java/training/alina/basic/gameleague/GameInfo.java

```diff
@@ -0,0 +1,166 @@
+package com.lftechnology.java.training.alina.basic.gameleague;
+
+/**
+ * Class GameInfo consists of game score informations of Manchester United
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class GameInfo {
+	private int noOfWins = 0;
+	private int noOfDraws = 0;
+	private int noOfDefeats = 0;
+	private int noOfGoalScored = 0;
+	private int noOfGoalConceded = 0;
+	private int noOfPoints = 0;
+	private String goalPoint = "";
+	private int firstScore = 0;
+	private int secondScore = 0;
+
+	public int getNoOfWins() {
+		return noOfWins;
+	}
+
+	public void setNoOfWins(int noOfWins) {
+		this.noOfWins = noOfWins;
+	}
+
+	public int getNoOfDraws() {
+		return noOfDraws;
+	}
+
+	public void setNoOfDraws(int noOfDraws) {
+		this.noOfDraws = noOfDraws;
+	}
+
+	public int getNoOfDefeats() {
+		return noOfDefeats;
+	}
+
+	public void setNoOfDefeats(int noOfDefeats) {
+		this.noOfDefeats = noOfDefeats;
+	}
+
+	public int getNoOfGoalScored() {
+		return noOfGoalScored;
+	}
+
+	public void setNoOfGoalScored(int noOfGoalScored) {
+		this.noOfGoalScored = noOfGoalScored;
+	}
+
+	public int getNoOfGoalConceded() {
+		return noOfGoalConceded;
+	}
+
+	public void setNoOfGoalConceded(int noOfGoalConceded) {
+		this.noOfGoalConceded = noOfGoalConceded;
+	}
+
+	public int getNoOfPoints() {
+		return noOfPoints;
+	}
+
+	public void setNoOfPoints(int noOfPoints) {
+		this.noOfPoints = noOfPoints;
+	}
+
+	/**
+	 * Function used to get the game scores
+	 * 
+	 * @param result
+	 *            {@link String} team and score sets
+	 * @return {@link String} manchester united scores
+	 * @author Alina Shakya <alinashakya@lftechnology.com>
+	 */
+	public String getGameScores(String result) {
+		String[] resultArray = result.split(" ");
+		int counter = 0;
+		int outputValue = 0;
+		for (int i = 0; i < resultArray.length; i++) {
+			try {
+				outputValue = Integer.parseInt(resultArray[i]);
+				if (counter == 0) {
+					firstScore = outputValue;
+					counter++;
+				} else {
+					secondScore = outputValue;
+				}
+			} catch (NumberFormatException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)